docs/en/install.md

```diff
@@ -125,6 +125,13 @@ The compatible MMTracking, MMCV, and MMDetection versions are as below. Please i
     pip install git+https://github.com/JonathonLuiten/TrackEval.git
     ```
 
+9. Install extra dependencies for TAO evaluation.
```

Comment:
please merge 8 and 9:
`8. Install extra dependencies:`
`- For MOTChallenge evaluation:`
`...`
`- For TAO evaluation:`
`...`